src/Makefile.test.include

```diff
@@ -185,7 +185,7 @@ DEFI_TEST_SUITE += \
 endif
 
 test_test_defi_SOURCES = $(DEFI_TEST_SUITE) $(DEFI_TESTS) $(JSON_TEST_FILES) $(RAW_TEST_FILES)
-test_test_defi_CPPFLAGS = $(AM_CPPFLAGS) $(DEFI_INCLUDES) $(TESTDEFS) $(EVENT_CFLAGS)
+test_test_defi_CPPFLAGS = $(AM_CPPFLAGS) $(DEFI_INCLUDES) $(TESTDEFS) $(EVENT_CFLAGS) -DTEST_BUILD
```

Comment:
Should we push this `TEST_BUILD` flag into code base? Maybe we just use it personally.

Reply:
I'll remove it.